template/dockerfiles/javascript/draft.yaml

```diff
@@ -5,11 +5,4 @@ nameOverrides:
 variables:
   - name: "PORT"
     description: "the port exposed in the application"
-    type: int
-  - name: "VERSION"
-    description: "the version of node used in the application"
-variableDefaults:
-  - name: "VERSION"
```

Comment:
do we still need to keep this VERSION variable?

Reply:
yeah i think this got wiped with that weird git issues before but I'll update it

Reply:
cool this also looks lik it impacts the gomodule ones